src/benchmark.rs

```diff
@@ -2,6 +2,7 @@
 //!
 //! This is useful for creating different benchmark on time of execution or
 //! the number of instruction for example.
+use core::usize;
```

Comment:
Not needed, I know sometimes Rust-Analyzer adds those without asking ^^'

Reply:
Oh that's a strange behavior from Rust-Analyzer haha